examples/pylab_examples/usetex_demo.py

```diff
@@ -1,70 +1,70 @@
 import matplotlib
 matplotlib.rc('text', usetex=True)
-import pylab
+import matplotlib.pyplot as plt
 import numpy as np
 
 # interface tracking profiles
 N = 500
 delta = 0.6
 X = np.linspace(-1, 1, N)
-pylab.plot(X, (1 - np.tanh(4.*X/delta))/2,    # phase field tanh profiles
+plt.plot(X, (1 - np.tanh(4.*X/delta))/2,    # phase field tanh profiles
            X, (X + 1)/2,                      # level set distance function
            X, (1.4 + np.tanh(4.*X/delta))/4,  # composition profile
            X, X < 0, 'k--',                   # sharp interface
            linewidth=5)
 
 # legend
-pylab.legend((r'phase field', r'level set', r'composition', r'sharp interface'), shadow=True, loc=(0.01, 0.55))
+plt.legend((r'phase field', r'level set', r'composition', r'sharp interface'), shadow=True, loc=(0.01, 0.55))
```

Comment:
while we are here fixing this file up, can you get rid of the extraneous "r's" in front of these strings? Also, let's put the keyword arguments on the next line.